packages/stream-model-instance/src/model-instance-document.ts

```diff
@@ -201,6 +204,21 @@ export class ModelInstanceDocument<T = Record<string, any>> extends Stream {
    */
   async patch(jsonPatch: Operation[], opts: UpdateOpts = {}): Promise<void> {
     opts = { ...DEFAULT_UPDATE_OPTS, ...opts }
+    jsonPatch.forEach((patch) => {
+      switch (patch.op) {
```

Comment:
do we need this switch statement since we already do the same thing in both cases?

Reply:
Yeah, we need it for the other operations. I couldn't figure out how to do `case 'add' | 'replace'` though.